### PR TITLE
Add `taskfile` validation hook

### PR DESCRIPTION
**Pull request by jrdnbradford**
Any interest in adding this hook? [`Task`](https://taskfile.dev/) is a cross-platform tool used to run tasks that is often used in CI, so it seems to fit with many of the hooks related to build pipelines in this repo.

### CODE REVIEW

**Comment by sirosen** [comment]
This seems great, and like you say, a good fit! Thanks for the contribution!
I need to make sure to read up on this to double-check the patterns used, etc, before merging, which will take me some time after my work day. Assuming no issues, I'll get this in and do a release soon.

**Comment by jrdnbradford** [comment]
@sirosen awesome, let me know if I can do anything else on this. On a personal note I've been migrating my `Makefile`s to `Taskfile`s and have been very happy with the results.

**Comment by jrdnbradford** [comment]
Awesome, glad to help with this project.

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -164,6 +164,23 @@
...
+  description: 'Validate Taskfile config against the schema provided by Task'
+  entry: check-jsonschema --builtin-schema vendor.taskfile
+  language: python
+  files: >
+    (?x)^(
+      Taskfile\.(yml|yaml)|
+      taskfile\.(yml|yaml)|
+      Taskfile\.dist\.(yml|yaml)|
+      taskfile\.dist\.(yml|yaml)
+    )$
```
[the remark this answers is not in the document]
Comment (jrdnbradford) [reply .pre-commit-hooks.yaml:181]: Yea, went more for readability here. One-liner seemed too much, one line for each option seemed overkill.

**Pull request merged**